src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.enums.Color;
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBERS_OF_FIGURES = 5;
+    private static final int DEFAULT_RADIUS_FOR_CIRCLE = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int type = random.nextInt(NUMBERS_OF_FIGURES);
+        double areaValue = random.nextDouble();
+        switch (type) {
+            case 0 :
+                return new Rectangle(colorSupplier.getRandomColor(), areaValue, areaValue);
+            case 1 :
+                return new Square(colorSupplier.getRandomColor(), areaValue);
+            case 2 :
+                return new RightTriangle(colorSupplier.getRandomColor(), areaValue, areaValue);
+            case 3 :
+                return new IsoscelesTrapezoid(colorSupplier.getRandomColor(), areaValue, areaValue);
+            case 4:
+                return new Circle(colorSupplier.getRandomColor(), areaValue);
+            default:
+                return null;

```

Comment:
Why null returned in default case? It is better to return default figure or throw exception